src/main/resources/bags.vm

```diff
@@ -4,7 +4,8 @@
       <link rel="stylesheet" type="text/css" href="/style.css">
   </head>
   <body>
-    <div class="breadcrumbs"><a href="/accessions">Accessions</a> <span class="hierarchy-separator" /><h1>$aname</h1></div>
+    <div class="breadcrumbs"><a href="/accessions">Accessions</a> <span class="hierarchy-separator" /></div>
+	<h1 style="padding: 10px;">$aname</h1>
```

Comment:
Same as above.  (in fact, the same CSS rule would cover this, thus reducing repetition)